### DISCUSSION TITLE
Ну что там, как, работы ведутся?

### DISCUSSION THREAD

**Discussion by kipkaev** [cdek-it/widget#37] [General]
Ждём 
> полнофункциональное ограничение региона с 4.0 версии

**Comment by vermorag** [comment]
Добрый день. В настоящий момент, в связи с загруженностью отделов разработки, срок выпуска 4 версии виджета определен II кварталом. Точнее подсказать не смогу.

**Comment by Andrew100500** [comment]
Стесняюсь спросить II кварталом какого года? Ну то что не 2024 это уже и ежу понятно. Так на 2025, 2026... 2030 на какой год ориентироваться?

**Comment by neverfan** [comment]
Есть какие-то новости по 4 версии? Очень много ограничений в 3 версии...

**Comment by neverfan** [comment]
Есть какие-то сроки? Есть ли смысл ждать в ближайшее время 4 версию?

**Comment by neverfan** [comment]
ping

**Comment by Andrew100500** [comment]
Вот еще год прошел, а "специалисты" СДЭКа продолжают сидеть на попе ровно и как сторожевые собаки защищать ошибки третье версии.

**Comment by uvb-it** [comment]
Есть доработанная 3 версия с загрузкой менее чем за секунду.
Кому интересно - пишите в тг @steveal84

**Comment by TiGR** [comment]
Уже осень 2025 года. @vermorag есть ли какие-то новости или проект забросили, полтора года прошло? Может хотя бы бету выпустить, наверняка нашлись бы желающие помочь с разработкой и тестированием.
